src/som/interpreter/actors/EventualMessage.java

```diff
@@ -341,4 +362,33 @@ public boolean isMessageReceiverBreakpointSet() {
   public void setIsMessageReceiverBreakpoint(final boolean triggerBreakpoint) {
     this.triggerMessageReceiverBreakpoint = triggerBreakpoint;
   }
+
+  public boolean isStepToReceiver() {
+    if (this.steppingType != null) {
```

Comment:
And why do you need all these helpers anyway?
And I still don't see why you can't just reuse the flags we already had for the breakpoints.